app/views/customers/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Customer Details</h1>
+
+<div id = "customer-name">
+    <p>Customer Name: <%= @customers.name %></p>
+</div>
+<div id = "item-list">
+    <ul>
+        <% @customers.items.each do |item|%>
+            <li><%= item.name %> - Price: <%= item.price %> - Supermarket: <%= item.supermarket.name %></li>
```

Comment:
Here on this <li> element, you can add another id so each item will have it's own css id that you can use in your testing. 
Good job iterating through all the items for the customer.